spec/models/abilities/common_spec.rb

```diff
@@ -259,6 +259,11 @@
       it { should_not be_able_to(:destroy, own_investment_in_selecting_budget) }
       it { should_not be_able_to(:destroy, own_investment_in_balloting_budget) }
 
+      it { should be_able_to(:edit, own_investment_in_accepting_budget) }
+      it { should_not be_able_to(:edit, own_investment_in_reviewing_budget) }
+      it { should_not be_able_to(:edit, own_investment_in_selecting_budget) }
+      it { should_not be_able_to(:edit, own_investment_in_balloting_budget) }
+
```

Comment:
Would it be possible to add a feature test as well, checking users can correctly click on the link to edit a form, fill in the form, and save the changes? Thanks a lot! :pray:

Reply:
yes, I think I can do it, are you using capybara for the features?

Reply:
Yes. I guess the right places to add the test would be `spec/features/budgets/investments_spec.rb`.